ide/swing.validation/external/simplevalidation-1.14.1-notice.txt

```diff
@@ -0,0 +1,3 @@
+ValidationAPI version 1.14.1
```

Comment:
An attempt to add copyright from here:
https://github.com/timboudreau/simplevalidation/blob/master/apache-license-2x.txt

Reply:
I'm inclined not to honor broken licenses. The Apache license explicitly states, that the file you have to look for as a user is NOTICE. If it is not there, I would ignore it, the user obiously was not interested in it either. There is also no NOTICE in the binary jar, further strengthening the point.

Reply:
Hmm, OK. I don't I like that (because the repository *has* a license file with the copyright), but I can probably live with this, at least for some time.

Reply:
If someone tells me he linceses his product under ALv2 and does not provide a NOTICE, I think the intention is obvious. If we assume that it is not ALv2, but an unnamed license derived from ALv2, then I suggest to make it "ALv2-simplevalidation" and use that, just as done with MIT.

Reply:
Ok. I've dropped this from this patch. I guess that while you may be technically right, being nice also may provides some advantages.